server/src/models/itemModel.js

```diff
@@ -0,0 +1,18 @@
+import { initDBConnection } from "../../db/db-connection.js";
+
+let db;
+
+export function initDb() {
+  db = initDBConnection();
```

Comment:
Does each model need to initialize the DB? I think you could initialize it once and just import it from db-connection.js

Reply:
I want to keep the model pure, without side effect

Reply:
That's a good goal 👍 

If you wanted it to be totally pure, you should pass the db connection into the model.